bin/chef-cluster

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
+#
+# Copyright 2017, SUSE
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmd=$1
+workercount=${2:-auto}
+vhostconfig=/etc/apache2/vhosts.d/chef-cluster.conf
+serverconfig=/etc/sysconfig/chef-server
+webroot=/srv/www/chef-cluster
+
+baseport=4001
+
+balancerstart=BEGIN-BALANCER-MEMBER
+balancerend=END-BALANCER-MEMBER
+
+usage() {
+  echo "usage: chef-cluster <enable|disable> [<workercount>|auto]"
+  exit 1
+}
+
+case $cmd in
+  enable)
+    [ "$workercount" == "auto" ] && workercount=$(grep -c ^processor /proc/cpuinfo)
+    [ "$workercount" -ge 2 ] || { echo "ERROR: workercount must be >= 2"; exit 2; }
+
+    # apache configuration
+    mkdir -p "$webroot"
+    a2enmod -q lbmethod_byrequests || a2enmod lbmethod_byrequests
+    test -e "$vhostconfig" || cat <<EOF > $vhostconfig
+# configuration generated by: chef-cluster script
+Listen 4000
+
+<VirtualHost *:4000>
+  DocumentRoot /srv/www/chef-cluster
+  ErrorLog /var/log/apache2/chef-cluster-error.log
+  CustomLog /var/log/apache2/chef-cluster-access.log combined
+
+  <Location />
+    Require all granted
+  </Location>
+
+  <Proxy balancer://chefworkers>
+    # $balancerstart
+    # $balancerend
+  </Proxy>
+
+  <Location /balancer-manager>
+    SetHandler balancer-manager
```

Comment:
The doc says this requires mod_status to be enabled, and I think it's disabled by default. Did you hit that?

Reply:
Nope. It worked fine without it.

Reply:
We can enable this module just in case if there are no drawbacks of doing so.